.gitignore

```diff
@@ -4,3 +4,4 @@
 Cargo.lock
 runtime/**/target
 .vscode
+.idea
```

Comment:
what is this for ?

Reply:
same reason for vscode. It's configuration file on Clion-Rust.